src/plugins/gres/common/gres_common.c

```diff
@@ -201,7 +201,7 @@ extern void common_gres_set_env(List gres_devices, char ***env_ptr,
 	ListIterator itr;
 	char *global_prefix = "", *local_prefix = "";
 	char *new_global_list = NULL, *new_local_list = NULL;
-	uint64_t tmp_gres_per_node = 0;
+	uint64_t tmp_gres_per_node = 0; int gres_per_task = 0;
```

Comment:
Put the second declaration on a separate line.

Reply:
Actually, drop the declaration, it is only used once? Just dereference the pointer where it is used in the info call below.